app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: :post_id
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  after_create :update_post_likes_counter
+
+  def update_post_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] OPTIONAL
Following best practices `update_likes_counter` should be marked as _private_ after implementing the `after_save` method.